src/Connection/PostObjects.php

```diff
@@ -362,12 +362,18 @@ public static function get_connection_config( $graphql_object, $args = [] ) {
 			unset( $connection_args['stati'] );
 		}
 
+		$from_field_name = lcfirst( $graphql_object->graphql_plural_name );
+
+		if ( $graphql_object->graphql_single_name === $graphql_object->graphql_plural_name ) {
+			$from_field_name = 'all' . ucfirst( $graphql_object->graphql_single_name );
```

Comment:
I think I'm mostly rubber-ducking here, not necessarily suggesting a specific change to this PR.

One thing that concerns me with the `all${SingleName}` convention is for `Type->Type` connections. 

Like: 

```graphql
{
  allCustomPostType {
    nodes {
       id
       title
       allCustomTaxonomy {
         nodes {
           id
           name
         }
       }
    }
  }
}
```

The `allCustomTaxonomy` reads weird to me when coming from a connection from another Type, because you're not _really_ asking for _all_ nodes of that Type, you're asking for selected nodes of that Type. . .I think. 🤔  

Naming things is hard.

I think this (`all{SingleName}`) pattern is a good improvement over what we have today though!

Reply:
Totally agree! We only want `all` prepended on RootQuery connections. Once you get into a Type->Type situation I think you're pretty safe from naming collisions between the single/plural, so we can just stick with the provided plural name as the field name.

Turns out I had achieved that for PostObject->TermObject connections but not the other way around! I took another look and think I've got a more targeted solution now. PR has been updated. The example you provided would now read:

```graphql
{
  allCustomTaxonomy {
    nodes {
      id
      name
      deer {
        nodes {
          id
          title
        }
      } 
    }
  }
  allCustomPostType {
    nodes {
       id
       title
       customTaxonomy {
         nodes {
           id
           name
         }
       }
    }
  }
}
```